src/js/constants/documentsProperties.json

```diff
@@ -399,7 +399,6 @@
         "properties": {
           "url": "dhei",
           "activities": [
-            "hiking",
```

Comment:
OK, although it should not be in this PR, and an issue should be created too, to track the change 🙂

Reply:
#2036